dev/benchmarks/macrobenchmarks/lib/main.dart

```diff
@@ -124,6 +126,13 @@ class HomePage extends StatelessWidget {
               Navigator.pushNamed(context, kColorFilterAndFadeRouteName);
             },
           ),
+          RaisedButton(
+            key: const Key(kFadingChildAnimationRouteName),
+            child: const Text('Fading Child Animation'),
```

Comment:
Same here, the button text "Fading Child Animation" doesn't match the "Filtered Child Animation" page title.

Reply:
See above comment about one page with multiple run modes...

Reply:
I adapted the page to change its title depending on the variant it is running.